web/src/routes/admin/submissions/[submissionId]/+page.svelte

```diff
@@ -119,7 +127,13 @@
 			<a href={'/admin/diff/' + data.id} class="btn btn-warning">Review Submission</a>
 		</div>
 	</div>
-{:else if data.state == 'Incorrect'}
+{:else if data.state == 'Incorrect' && data.stateReason == 'BuildError'}
+	<h3 style="text-align:center">Build Output</h3>
+	<textarea use:stretchTextarea class="code mb-3 form-control" disabled>{data.stateReasonDetails}</textarea>
+{:else if data.state == 'Incorrect' && data.stateReason == 'TimeLimitExceeded'}
+	<h3 style="text-align:center">Details</h3>
+	<textarea use:stretchTextarea class="code mb-3 form-control" disabled>{data.stateReasonDetails}</textarea>
+{:else}
```

Comment:
Now the diff shows even if it was marked Correct (since it might not actually match fully, and is thus worth showing)